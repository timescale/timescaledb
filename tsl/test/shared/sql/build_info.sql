-- This file and its contents are licensed under the Timescale License.
-- Please see the included NOTICE for copyright information and
-- LICENSE-TIMESCALE for a copy of the license.

-- Check what get_git_commit returns
SELECT pg_typeof(commit_tag) AS commit_tag_type,
       pg_typeof(commit_hash) AS commit_hash_type,
       length(commit_hash) AS commit_hash_length,
       pg_typeof(commit_time) AS commit_time_type
  FROM _timescaledb_functions.get_git_commit();

SELECT extname, obj_description(oid, 'pg_extension') FROM pg_extension WHERE extname = 'timescaledb';

