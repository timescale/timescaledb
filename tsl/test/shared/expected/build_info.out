-- This file and its contents are licensed under the Timescale License.
-- Please see the included NOTICE for copyright information and
-- LICENSE-TIMESCALE for a copy of the license.
-- Check what get_git_commit returns
SELECT pg_typeof(commit_tag) AS commit_tag_type,
       pg_typeof(commit_hash) AS commit_hash_type,
       length(commit_hash) AS commit_hash_length,
       pg_typeof(commit_time) AS commit_time_type
  FROM _timescaledb_functions.get_git_commit();
 commit_tag_type | commit_hash_type | commit_hash_length |     commit_time_type     
-----------------+------------------+--------------------+--------------------------
 text            | text             |                 40 | timestamp with time zone
(1 row)

SELECT extname, obj_description(oid, 'pg_extension') FROM pg_extension WHERE extname = 'timescaledb';
   extname   |                                    obj_description                                    
-------------+---------------------------------------------------------------------------------------
 timescaledb | Enables scalable inserts and complex queries for time-series data (Community Edition)
(1 row)

